connection.cpp

```diff
@@ -159,12 +202,12 @@ void Connection::leaveRoom(Room* room)
     job->start();
 }
 
-void Connection::getMembers(Room* room)
-{
-    RoomMembersJob* job = new RoomMembersJob(d->data, room);
-    connect( job, &RoomMembersJob::result, d, &ConnectionPrivate::gotRoomMembers );
-    job->start();
-}
+//void Connection::getMembers(Room* room)
+//{
+//    RoomMembersJob* job = new RoomMembersJob(d->data, room);
+//    connect( job, &RoomMembersJob::result, d, &ConnectionPrivate::gotRoomMembers );
+//    job->start();
+//}
```

Comment:
The short answer is dunno. Long answer: I'm still unsure if it might be useful sometime in the future or we are supposed to get room members through syncs exclusively. Personally, I feel /sync to be quite heavy at times and prefer some simpler requests, like this one. But if this one is not synced with /sync then we don't have much choice.
Case in point: imagine we've got a room directory and I can look at room properties before joining it. In particular, browse through the list of participants. Definitely not a case for /sync.